tests/docker/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+# vim: set ft=ansible:
+#
+# !!!NOTE!!! This playbook was tested using Ansible 2.1; it is recommended
+# that the same version is used.
+#
+# This playbook tests the basic functions of docker or docker-latest. These
+# functions include:
+#   - docker build
+#   - docker images
+#   - docker ps
+#   - docker pull
+#   - docker rm
+#   - docker rmi
+#   - docker run
+#   - docker start
+#   - docker stop
+#
+# To test docker-current, set the g_docker_latest variable to false in the
+# vars.yml file.
+#
+
+- name: Docker - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+
+  pre_tasks:
+    - name: Check for docker-latest package
+      command: rpm -qa
+      register: pkgs
+
+    - name: Fail if docker latest is not installed
+      fail:
+        msg: "This test is not valid because docker-latest is not installed"
+      when: "'docker-latest' not in pkgs.stdout"
```

Comment:
Good catch!  I totally  missed that.  Hopefully it should be good now.  I ran a quick test and it seems to logically work.